### ISSUE TITLE
Use utils.DisableLogger() to reduce dynspinup output in runs

### ISSUE THREAD

**Issue by fmaussion** [OGGM/oggm#1476]
the output is too wordy for the standard workflow - lets discuss

**Comment by pat-schmitt** [comment]
Oh, that's great that this context manager exists. I can include it inside the dynamic mu calibration to suppress the massages of the inversion at each iteration. I assume it works like this:
```python
with utils.DisableLogger():
    calibrate_inversion_from_consensus(gdirs, ...)
```

**Comment by pat-schmitt** [comment]
I switched off the logging of intermediate steps during the dynamic mu star calibration in https://github.com/OGGM/oggm/pull/1488.

Would it be a good idea to add the outcome of the dynamic mu star calibration to the gdir.log (`full success`, `part success` or `fixed geometry`)? Currently, you only can see the outcome in the glacier statistics after calling `utils.compile_glacier_statistics`.
